### PR TITLE
Ability to tag new tables, Resolves #233

### LINKED ISSUE #233: Ability to specify tags to a new table
When running credstash setup it would be great to be able to apply custom tags in addition to the default `Name=Credstash` tag.

### PR DESCRIPTION
**Pull request by corrjo**
adds a `--tags` to `credstash setup`

Can be used as `credstash -t some-new-table setup --tags MyTag=MyValue AnotherTag=AnotherValue`

### CODE REVIEW

File: credstash.py [hunk 1]
```diff
@@ -581,6 +581,16 @@ def createDdbTable(region=None, table="credential-store", **kwargs):
         ]
     )
 
+    if tags:
+        tagset = []
+        for i in tags:
+            tag = i.split('=')
```
Comment (mike-luminal) [inline credstash.py]: Can you add some input validation here? 

We want to raise a friendly error if the tag input doesn't match the expected format.

Comment (corrjo) [reply credstash.py]: Good idea. Just saw we already have `key_value_pair()` which we are using for type validation with argparse. I'll make sure `--tags` is enforcing it also.

Comment (corrjo) [reply credstash.py]: Should be good now. Let me know if there is anything else :)

**Comment by corrjo** [comment]
@mike-luminal Let me know if you need anything else, and let me know if you ever need more maintainers, I'd be happy to help!

**Pull request merged**